Index: src/LMS_start.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/LMS_start.java	(revision adb01ecb4fd9459e3084386113288147a2723c56)
+++ src/LMS_start.java	(date 1632945548352)
@@ -230,7 +230,7 @@
                 System.out.println(id + " " + name + " " + author + " " + genre);
             }
 
-            System.out.println("Өзгерту үшін таңдаңыз");
+            System.out.println("Өзгерту үшін таңдаңыз...");
             int id = scnDigit.nextInt();
             if(renameType.equalsIgnoreCase("name")){
                 updateSql = "update booklist set namebook=?  where id=" + id;
